containers/base/Dockerfile

```diff
@@ -183,9 +183,8 @@ RUN echo "deb-src http://ftp.us.debian.org/debian testing main" >> /etc/apt/sour
 # Add some unchanging bits - specifically node modules (that need to be kept in sync
 # with packages.json manually, but help save build time, by preincluding them in an
 # earlier layer).
-# Note: ws is now over 1.0 but using that gives issues so leaving at 0.4.2 for now.
```

Comment:
This is worrying. @qimingj or @chmeyers, do you guys have context about what sort of issues were seen with the update? If not we'll have to do some testing.

Reply:
I think @nikhilk did that and added that comments. I am fine with it with some manual testing.

Reply:
gramster@ originally added that comment in https://github.com/googledatalab/datalab/commit/e8659d9b6124f9a17c4ffedb4f749cbb8c257db2 (April of 2016)

Reply:
FTR I did manually test with this change through the socket.io codepath and things seem to be more robust (see b/ link in original PR).

Reply:
Maybe that explains why we had some stability issues with socket io.

Reply:
fischman@: Do you know if we are going to hit any of the "Backwards Incompatible Changes" listed in http://channels.readthedocs.io/en/latest/releases/1.0.0.html ?

If we're not sure, I'm okay with upgrading it and fixing any issues we find.

Reply:
@chmeyers I don't _know_, but I expect this to be all fine as the motivating ws issue contains evidence of people using socket.io 1.7.x with ws 1.1.x (and we don't use ws directly).

Reply:
Okay.  Update Dockerfile.py2 too, and I'll approve/merge.

Reply:
Done.